docs/turbo.json

```diff
@@ -14,12 +14,17 @@
       "dependsOn": ["^build", "rss", "schema"]
     },
     "dev": {
-      "dependsOn": ["schema"]
+      "dependsOn": ["schema"],
+      "persistent": true,
+      "cache": false
     },
     "schema": {
+      "inputs": [], // None of the files in this workspace matter
```

Comment:
This empty array setting _should_ work, but I'm seeing that in `--dry=json` and `--summarize` it still takes the full workspace. Not a blocker for this PR, but it's a bug we should fix